geoindicators/src/test/groovy/org/orbisgis/geoclimate/geoindicators/SpatialUnitsTests.groovy

```diff
@@ -225,4 +227,128 @@ class SpatialUnitsTests {
         def countRows = postGIS.firstRow "select count(*) as numberOfRows from $outputTable"
         assert 100 == countRows.numberOfRows
     }
+
+    @Test
+    void sprawlAreasTest1() {
```

Comment:
Is in one test for sprawling you test the "collapsing distance" to remove some objects being long but so thin that they disappear ?

Reply:
yep